.golangci.yml

```diff
@@ -38,10 +38,17 @@ linters:
 issues:
   max-same-issues: 0
   max-issues-per-linter: 0
+  exclude-use-default: false
```

Comment:
What about adding a comment here:
> We are disabling default golangci exclusions because we want to help reviewers to focus on reviewing the most relevant changes in PRs and not too much on nitpicking.